roles/fio_distributed/templates/server_vm.yml.j2

```diff
@@ -13,52 +13,53 @@ spec:
       cores: {{ workload_args.vm_cores | default(1) }}
     devices:
       disks:
-        - name: registrydisk
-        - name: cloudinitdisk
+        - disk:
+            bus: virtio
+          name: registrydisk
+        - disk:
+            bus: virtio
+          name: cloudinitdisk
 {% if workload_args.storageclass is defined
     or workload_args.hostpath is defined %}
         - disk:
             bus: virtio
           name: data-volume
           serial: data
+{% else %}
+        - disk:
+            bus: virtio
+          name: emptydisk
 {% endif %}
     resources:
       requests:
         memory: {{ workload_args.vm_memory | default('5G') }}
   volumes:
     - name: registrydisk
       containerDisk:
-        image: {{ workload_args.vm_image | default('kubevirt/fedora-cloud-container-disk-demo:latest') }}
+        image: {{ workload_args.vm_image | default('quay.io/kubevirt/fedora-cloud-container-disk-demo:latest') }}
     - name: cloudinitdisk
       cloudInitNoCloud:
         userData: |-
           #cloud-config
-          password: fedora
+          password: ripsaw
```

Comment:
this may not be relevant to this PR but in general wouldn't it be easier to have the user supply a public key to enable login rather than a password?  These are so guessable.

Reply:
Good suggestion, we could refactor this for all VM templates.. will take a look.